src/pocketmine/block/Liquid.php

```diff
@@ -409,29 +416,54 @@ private function getSmallestFlowDecay(Vector3 $pos, $decay){
 
 		return ($decay >= 0 && $blockDecay >= $decay) ? $decay : $blockDecay;
 	}
-
-	private function checkForHarden(){
+	
+	public function checkForHarden(){ //Performance improvement (Not that much, though!)
```

Comment:
Nein, die funktion a sich is ein performance improvement
Schau dir mal den neusten commit an